generation/WinSDK/enums.json

```diff
@@ -30873,5 +30873,598 @@
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUICONVERTA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUICONVERTW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "CHANGE_ICON_FLAGS",
+    "type":"int",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "CIF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUICHANGEICONA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUICHANGEICONW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "CHANGE_SOURCE_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "CSF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUICHANGESOURCEA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUICHANGESOURCEW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "EDIT_LINKS_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "ELF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIEDITLINKSA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIEDITLINKSW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "INSERT_OBJECT_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "IOF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIINSERTOBJECTA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIINSERTOBJECTW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "OBJECT_PROPERTIES_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "OPF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIOBJECTPROPSA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIOBJECTPROPSW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "VIEW_OBJECT_PROPERTIES_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "VPF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIVIEWPROPSA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIVIEWPROPSW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "FADF",
+    "type": "ushort",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "FADF_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "SAFEARRAY",
+        "field": "fFeatures"
+      }
+    ]
+  },
+  {
+    "name": "DISPID",
+    "type": "int",
+    "autoPopulate": {
+      "filter": "DISPID_",
+      "header": "olectl.h"
+    },
+    "members": [
+      {
+        "name": "MEMBERID_NIL",
+        "value": "(DISPID_UNKNOWN)"
```

Comment:
But can't users create any DISPID's they want to? I'm not sure it makes sense to turn DISPID into an enum since it's an open-ended thing.

Reply:
I am unsure. It's a question I raised in the issue. same thing around memid.